pom.xml

```diff
@@ -64,6 +64,12 @@
       <artifactId>google-http-client</artifactId>
       <version>1.23.0</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.5</version>
+    </dependency>
```

Comment:
So I'm curious, we're using Gson but we already have Json serialization mechanisms on in-toto no? wouldn't it be useful to use the same? (i.e., double check we are using the same library for json serialization throughout here and on in-toto-java)

Reply:
I wonder if this is just a local issue, because `mvn package` failed before I included it...

```
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/transport/GrafeasTransport.java:158: Undefined reference: com.google.gson.Gson                                                                                      
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/transport/GrafeasTransport.java:158: Undefined reference: void com.google.gson.Gson.<init>()                                                                        
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/transport/GrafeasTransport.java:159: Undefined reference: String com.google.gson.Gson.toJson(Object)                                                                
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:232: Undefined reference: com.google.gson.Gson                                                                                                   
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:232: Undefined reference: void com.google.gson.Gson.<init>()                                                                                     
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:236: Undefined reference: Object com.google.gson.Gson.fromJson(String, java.lang.reflect.Type)                                                   
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:233: Undefined reference: void com.google.gson.reflect.TypeToken.<init>()                                                                        
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:329: Undefined reference: com.google.gson.Gson                                                                                                   
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:329: Undefined reference: void com.google.gson.Gson.<init>()                                                                                     
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:331: Undefined reference: Object com.google.gson.Gson.fromJson(String, Class)                                                                    
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:287: Undefined reference: com.google.gson.Gson                                                                                                   
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:287: Undefined reference: void com.google.gson.Gson.<init>()                                                                                     
[ERROR] /home/saky/Work/secure-systems-lab/in-toto-jenkins-plugin/src/main/java/io/jenkins/plugins/intotorecorder/InTotoWrapper.java:288: Undefined reference: String com.google.gson.Gson.toJson(Object)                                                                             
```

Also:

```
[ERROR] Failed to execute goal org.codehaus.mojo:animal-sniffer-maven-plugin:1.15:check (check) on project in-toto: Signature errors found. Verify them and ignore them with the proper annotation if needed. -> [Help 1]
```

Reply:
Removed, and it's passing...